tests/conf.js

```diff
@@ -19,7 +19,11 @@ const baseConfig = {
   //Enabling specs as they are working.
   suites: {
     // e2e:'e2e/**/*.js',
-    e2e: ['**/login.spec.js'],
+    e2e: [
+      '**/login.spec.js',
+      '**/report-date-filter.js',
+      'tests/e2e/reports-subject.js'
```

Comment:
Please update with the trailing comma as you did in the update to your other PR.

```suggestion
      'tests/e2e/reports-subject.js',
```